test/tools/libtesteth/TestHelper.h

```diff
@@ -167,12 +167,14 @@ json_spirit::mObject fillJsonWithTransaction(eth::Transaction const& _txn);
 
 //Fill Test Functions
 int createRandomTest(std::vector<char*> const& _parameters);
-void doTransactionTests(json_spirit::mValue& _v, bool _fillin);
-void doStateTests(json_spirit::mValue& v, bool _fillin);
-void doVMTests(json_spirit::mValue& v, bool _fillin);
-void doBlockchainTests(json_spirit::mValue& _v, bool _fillin);
-void doBlockchainTestNoLog(json_spirit::mValue& _v, bool _fillin);
-void doTransitionTest(json_spirit::mValue& _v, bool _fillin);
+//do*Tests(_v, _fillin) always return a filled test.
+//When _fillin is true, _v is supposed to contain a filler.  Otherwise, _v is also a filled test.
+json_spirit::mValue doTransactionTests(json_spirit::mValue const& _v, bool _fillin);
+json_spirit::mValue doStateTests(json_spirit::mValue const& v, bool _fillin);
+json_spirit::mValue doVMTests(json_spirit::mValue const& v, bool _fillin);
+json_spirit::mValue doBlockchainTests(json_spirit::mValue const& _v, bool _fillin);
+json_spirit::mValue doBlockchainTestNoLog(json_spirit::mValue const& _v, bool _fillin);
+json_spirit::mValue doTransitionTest(json_spirit::mValue const& _v, bool _fillin);
 void doRlpTests(json_spirit::mValue& v, bool _fillin);
```

Comment:
Oh I forgot to rename these names.

I'll change `doRlpTests` in a separate PR.